sonar-kotlin-plugin/src/main/resources/static/documentation.md

```diff
@@ -12,6 +12,22 @@ key: kotlin
 
 You can discover and update Kotlin-specific [properties](/analysis/analysis-parameters/) in:  <!-- sonarcloud -->Project <!-- /sonarcloud -->**[Administration > General Settings > Kotlin](/#sonarqube-admin#/admin/settings?category=kotlin)**.
 
+## Kotlin Analysis and Bytecode
+If you are not using the Maven or Gradle Sonar plugins, it is strongly recommended to provide the paths of all dependency binaries used by
+the project in order to improve the analysis accuracy. You can provide these using the `sonar.java.libraries` property. This is a list of 
+comma-separated paths to files with third-party libraries (JAR or Zip files) used by your project. Wildcards can be used: 
+`sonar.java.libraries=path/to/Library.jar,directory/**/*.jar`
+
+Note that if you use the Gradle or Maven Sonar plugins to scan your code, these plugins will usually auto-detect the value for the property.
+
+## Specifying the Kotlin Source Code Version
+You can explicitly define which Kotlin version the analyzer should analyze your code based on. Provide the desired version in the format
+`X.Y` as value to the `sonar.kotlin.source.version` property (e.g. `1.7`).
+
+## Multi-Threaded Parsing
+As of August 2022, the SonarKotlin analyzer supports multi-threaded parsing. This is an experimental feature and is disabled by default.
```

Comment:
Note: "SonarKotlin": there is no such product existing :-)

As of August 2022, the Kotlin analyzer ...